napari/_qt/utils.py

```diff
@@ -123,3 +124,17 @@ def qt_signals_blocked(obj):
     obj.blockSignals(True)
     yield
     obj.blockSignals(False)
+
+
+def find_ancestor_mainwindow(widget):
```

Comment:
Although this is a cute function, is this needed anymore?

Reply:
removed